### DISCUSSION TITLE
PowerBI connection to TruBudget via API

### DISCUSSION THREAD

**Discussion by RMutius** [openkfw/TruBudget#1249] [Ideas]
It would be great to have the opportunity to connect TruBudget with PowerBI via an API connection. This would allow evaluating and visualising workflow information as well as their status. However, for a smooth functioning this API connection would require a dynamic token generation in PowerBI in order to be able to set an automatic dataset refresh.

**Comment by jzakotnik** [comment]
The connection was successfully established and documentation included.

**Comment by Stezido** [comment]
Added in #1250
